src/main/java/com/softserve/demo/service/impl/OrderServiceImpl.java

```diff
@@ -17,24 +17,41 @@
 @Transactional
 public class OrderServiceImpl implements OrderService {
 
-    private OrderRepository orderRepository;
-    private LocationRepository locationRepository;
+    private static final String CONTRACT_WAS_CREATED = "New contract was created";
+    private static final String CONTRACT_WAS_UPDATED = "Contract was updated";
+    private static final String CONTRACT_WAS_DELETED = "Contract was deleted";
+    private static final String BY = " by %s %s";
+    private static final String WITH = " with %s";
+    private static final String TEXT_CREATE_CONTRACT = "Dear %s,\n" +
```

Comment:
I would prefer to move such constants into separate class like MessageConstants or something.